vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow-integration-tests/src/main/java/com/vaadin/flow/component/richtexteditor/tests/RichTextEditorAsDeltaBinderPage.java

```diff
@@ -0,0 +1,70 @@
+package com.vaadin.flow.component.richtexteditor.tests;
+
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.NativeButton;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.richtexteditor.RichTextEditor;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.binder.ValidationException;
+import com.vaadin.flow.router.Route;
+
+@Route("vaadin-rich-text-editor/as-delta-binder")
+public class RichTextEditorAsDeltaBinderPage extends Div {
```

Comment:
Covers the previous binder tests from `RichTextEditorAsDeltaIT`.

`RichTextEditorAsDeltaPage` / `RichTextEditorAsDeltaIT` have been removed, as the remaining value related tests should be covered by `RichTextEditorValueIT`.